### DISCUSSION TITLE
Translations file missing from production build

### DISCUSSION THREAD

**Discussion by mohil-nft** [adonisjs/core#4430] [Help] [answered]
### Package version

  "@adonisjs/i18n": 1.6.0

What happens to the translation files inside `resources/lang/en`  when we run  `node ace build --production`. I'm unable to get the translations while running it on production server since they are not there in the build folder.

**Comment by Julien-R44** [comment, marked as answer]
Make sure to add your translation files to your meta files in the adonisrc file, so that they get copied when building. 

See https://docs.adonisjs.com/guides/adonisrc-file#metafiles

**Reply by mohil-nft** [reply]
@Julien-R44 Thanks! I couldn't find this in v5 docs.
